src/librustc_trans/lib.rs

```diff
@@ -66,6 +66,7 @@ extern crate rustc_errors as errors;
 extern crate serialize;
 #[cfg(windows)]
 extern crate cc; // Used to locate MSVC
+pub extern crate tempdir;
```

Comment:
why "pub extern crate"?

Reply:
Ah ok you are exporting it for tests so that they don't need to use Cargo. Could you add a comment to that effect?